app/shared/api/blockchain/extrinsic/constants.ts

```diff
@@ -0,0 +1,24 @@
+import { type ApiPromise } from '@polkadot/api';
+import { type SubmittableExtrinsic } from '@polkadot/api/types';
+
+import { type TransactionType } from '../types';
+
+export const SUBMITTABLE_EXTRINSICS: Record<
+  TransactionType,
+  (args: Record<string, any>, api: ApiPromise) => SubmittableExtrinsic<'promise'>
+> = {
+  TRANSFER: ({ dest, value }, api) => {
+    return api.tx.balances['transferKeepAlive']
+      ? api.tx.balances['transferKeepAlive'](dest, value)
+      : api.tx.balances['transfer'](dest, value);
+  },
+  TRANSFER_STATEMINE: ({ dest, value, asset }, api) => {
+    return api.tx.assets.transfer(asset, dest, value);
+  },
+  TRANSFER_ORML: ({ dest, value, asset }, api) => {
+    return api.tx['tokens']['transfer'](dest, asset, value);
+  },
+  TRANSFER_ALL: ({ dest }, api) => {
```

Comment:
This approach has proven to be uncalable and unflexible in Spektr. Moving towards interface + implementations + factory is good, but why leave this global map? Those two approaches contradict each other: one tries to hide a concerete implementation whereas other makes everyone aware of them

Reply:
I've got no intent to refactor this part during the refactoring
This code was moved from the other file